tests/ui/conftest.py

```diff
@@ -50,23 +50,24 @@ def jwt_secret(base_url, json_file):
 
 
 @pytest.fixture
-def initial_data(transactional_db, live_server, base_url):
+def initial_data(transactional_db, live_server):
     call_command('generate_addons', 10, app='firefox')
 
 
 @pytest.fixture
-def create_superuser(transactional_db, live_server, base_url):
+def create_superuser(transactional_db, live_server, base_url, tmpdir):
     hostname = urlparse.urlsplit(base_url).hostname
     create_switch('super-create-accounts')
     call_command('loaddata', 'initial.json')
+    temp = tmpdir.join('variables.json')
 
     call_command(
         'createsuperuser',
         interactive=False,
         username='uitest',
         email='uitester@mozilla.org',
         add_to_supercreate_group=True,
-        save_api_credentials='tests/ui/variables.json',
+        save_api_credentials=str(temp),
```

Comment:
We should also read the contents and add them to the `variables` fixture so we don't need the `json_file` fixture.